app/locale/en_US/template/email/password_new.html

```diff
@@ -16,8 +17,8 @@
     <tr>
         <td class="action-content">
             <h1>{{htmlescape var=$customer.name}},</h1>
-            <p><strong>Password</strong>: (the password you set when creating your account) </p>
-            <p>You can change your password at any time by logging into <a href="{{store url="customer/account/"}}">your account</a>.</p>
+            <p><strong>Your new password is:</strong> {{htmlescape var=$customer.password}}</p>
+            <p>As the password is in plaintext, you should set a new password as soon as possible by logging into <a href="{{store url="customer/account/"}}">your account</a>.</p>
```

Comment:
Please note the password was sent in the plaintext format. You must set immediately a new password into your <account>.